MAINTAINERS.md

```diff
@@ -7,5 +11,3 @@
 | Sayali Gaikawad  | [gaiksaya](https://github.com/gaiksaya)             | Amazon      |
 | Abhinav Gupta    | [abhinavGupta16](https://github.com/abhinavGupta16) | Amazon      |
```

Comment:
Can you please remove Abhinav from here since he is not a maintainer anymore?
Thanks!

Reply:
Will do it in another PR.